doxia-decoration-model/pom.xml

```diff
@@ -36,16 +36,22 @@ under the License.
 
   <dependencies>
     <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-component-annotations</artifactId>
+      <groupId>org.eclipse.sisu</groupId>
+      <artifactId>org.eclipse.sisu.plexus</artifactId>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
```

Comment:
Should this be provided?

Reply:
I must admit I have quite a hard time to understand the `provided` scope which seems to depend on how you suppose the module will be used.  In this case, I don't know much about doxia, so I'll defer to you.  Fwiw, the plexus annotations were in the default scope, so I used the same.

Reply:
Let's leave it as-is for now.